apps/src/javalab/types.ts

```diff
@@ -0,0 +1,7 @@
+export const JavalabEditorDialog = {
```

Comment:
Will constants in javalab/constants.js also be moved to this file?

Reply:
We probably should! But for now I'm leaving this scoped to typescript types we are actively using.